hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/stream/DirectoryServerSource.java

```diff
@@ -41,18 +42,19 @@ public DirectoryServerSource(Path root) {
    * @param id name of the subdirectory to replitace relative to root.
    */
   public Map<String, Path> getFilesToStream(String id)
-      throws InterruptedException {
+          throws InterruptedException {
     Map<String, Path> files = new HashMap<>();
     final Path streamingDir = root.resolve(id);
     try {
-      Files.walk(streamingDir)
-          .filter(Files::isRegularFile)
-          .forEach(path -> {
-            files.put(root.relativize(path).toString(), path);
-          });
+      try (Stream<Path> list = Files.walk(streamingDir)
```

Comment:
Can you please merge these two `try` blocks?

Reply:
Thanks @adoroszlai for reviewing, I have fixed it.